README.md

```diff
@@ -88,21 +88,11 @@ You can sign up [here](https://rubyforgood.slack.com/join/shared_invite/zt-1kfei
      - Do you develop on Windows? We'd love to hear (and for you to submit a PR explaining) how you do it. 🙏🏻 
    - Create a `database.yml` file on `config/` directory with your database configurations. You can also copy the existing files called [`database.yml.example`](config/database.yml.example) and [`.env.example`](.env.example) and change the credentials.
 
-### 3. Run Setup
-Run the following command should run all the neccessary steps to get up and running.
-```
-bin/setup
-```
-
-### 4.  Run the application
-Run the following command to start the server locally.
-```
-bin/start
-```
-
-You should be able to open up a browser and goto http://localhost:3000/ and see the human essentials page.
+   </br>
 
-### 5. Login as a sample user
+3. Run `bin/setup`
```

Comment:
Maybe a note about each of these for what they do or how often you should run them?

Reply:
This seems unnecessary to me because that information wasn't here before. I think the terminal output is enough for detail on what it does and the sequential steps _implies_ that it should only be ran once.